pkg/backup/client.go

```diff
@@ -357,6 +359,7 @@ func (bc *Client) backupRange(
 		StorageBackend: bc.backend,
 		RateLimit:      rateLimit,
 		Concurrency:    concurrency,
+		IsRawKv:        isRawKv,
 	}
 	push := newPushDown(ctx, bc.mgr, len(allStores))
 
```

Comment:
It seems you need to set the `IsRawKv` field of `backupMeta` below here.

Reply:
[Done](https://github.com/pingcap/br/pull/101/files#diff-7dfd05fbd1b9304c3d90dc85e3ad8090R383)